frontend/components/AddSite/TemplateSiteList/index.js

```diff
@@ -60,31 +59,24 @@ export class TemplateList extends React.Component {
     const cellsPerRow = Math.min(templateKeys.length, MAX_CELLS_PER_ROW);
     // generate a two-dimensional array of template keys
     const templateRows = createRowsOf(templateKeys, cellsPerRow);
-    // i.e. 'whole', 'half', 'third', or 'fourth'
-    const cellSize = CELL_WIDTHS[cellsPerRow];
 
     let index = 0;
 
     const templateGrid = templateRows.map(row => (
-      <div className="usa-grid" key={row}>
+      <div className="flexbox-grid flexbox-grid-4" key={row}>
```

Comment:
Looks like the rows now are meant to always contain 4 items. Previously, there was (and still remains) a bunch of a logic to allow for "easily" adjusting the number of items (that's what the `MAX_CELLS_PER_ROW` variable was for).

I'm fine with sticking with 4 per row. What happens though when we add a 5th template? Then we'll have a full row of 4 and another row of just 1.

If we do stick with a static 4-per-row, we should clean up the remaining logic in this file. @line47 don't worry about doing that -- I can do it depending on what we decide.

Reply:
Alternatively, we could change this line to 

```jsx
<div className={`flexbox-grid flexbox-grid-${MAX_CELLS_PER_ROW}`} key={row}>
```

and then make a few more `.flexbox-grid-#` classes in `_flexbox.scss`.

Let's discuss and handle in another issue/PR though.

Reply:
I like the `flexbox-grid-#` idea a lot! Let's do that... want to do in a separate PR then @jseppi ?

Reply:
yeah, we can file an issue and handle it separately 👍

Reply:
filed https://github.com/18F/federalist/issues/1298